src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/LoggerMessageDefineAnalyzer.cs

```diff
@@ -247,29 +247,8 @@ private void AnalyzeFormatArgument(OperationAnalysisContext context, IOperation
 
             switch (argumentExpression)
             {
-                case ILiteralOperation { ConstantValue: { HasValue: false, Value: string constantValue } }:
+                case IOperation { ConstantValue: { HasValue: true, Value: string constantValue } }:
```

Comment:
This will handle constant interpolated strings only for compiler versions that support it. I'm not sure whether this should support old versions or not.